tools/accuracy_checker/accuracy_checker/adapters/centernet.py

```diff
@@ -0,0 +1,161 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+from numpy.lib.stride_tricks import as_strided
+
+from ..adapters import Adapter
+from ..config import StringField
+from ..preprocessor import CenterNetAffineTransformation
+from ..representation import DetectionPrediction
+
+
+class CTDETAdapter(Adapter):
+    __provider__ = 'ctdet'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update(
+            {
+                'hm': StringField(description="Object center points heatmap."),
```

Comment:
please give more descriptive name for parameters, if they mean output layer name, please add postfix `_out`